test/functional/rpc_psbt.py

```diff
@@ -59,6 +59,9 @@ def set_test_params(self):
             ["-walletrbf=0", "-changetype=legacy"],
             []
         ]
+        # whitelist peers to speed up tx relay / mempool sync
```

Comment:
feel like there should be a subroutine to do this for any given test

Reply:
also above: "#TODO: Remove address type restrictions once taproot has psbt extensions" can this be dropped now?